napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -64,27 +95,50 @@ def __init__(
         enabled: bool = True,
         installed: bool = False,
         npe_version=1,
+        versions_conda: list = [],
+        versions_pypi: list = [],
     ):
         super().__init__(parent)
+        self.url = url
+        self._versions = {}
+        self._versions_conda = versions_conda
+        self._versions_pypi = versions_pypi
         self.setup_ui(enabled)
         self.plugin_name.setText(package_name)
+
+        if len(versions_pypi) > 0:
+            self._populate_version_dropdown('PyPI')
+        else:
+            self._populate_version_dropdown('Conda')
+
         self.package_name.setText(version)
-        self.summary.setText(summary)
-        self.package_author.setText(author)
+        if summary:
+            self.summary.setText(summary + '\br')
```

Comment:
This line is adding an `r` to every description lol, we should probably remove this?